consul/README.md

```diff
@@ -1,32 +1,143 @@
 # Consul Integration
 
-## Overview
+# Overview
 
-Get metrics from consul service in real time to:
+The Datadog Agent collects many metrics from Consul nodes, including those for:
 
-* Visualize and monitor consul states
-* Be notified about consul failovers and events.
+* Total Consul peers
+* Service health - for a given service, how many of its nodes are up, passing, warning, critical?
+* Node health - for a given node, how many of its services are up, passing, warning, critical?
+* Network coordinates - inter- and intra-datacenter latencies
```

Comment:
Let avoid bullet points as they dont fit well in all the locations we plan to use this content on the website and in app.